backend/package.json

```diff
@@ -1,17 +1,15 @@
 {
-  "name": "HangHub-backend",
+  "name": "hang-hub-backend",
```

Comment:
I think it should be `hanghub-backend`. We do not use HangHub as 2 separate words. It is like "CKEditor". We use "ckeditor" when we need to use all lowwer case, not "c-k-editor".